src/NServiceBus.Core/MessageMutators/MutateTransportMessage/MutateIncomingTransportMessageBehavior.cs

```diff
@@ -19,11 +25,21 @@ public Task Invoke(IIncomingPhysicalMessageContext context, Func<IIncomingPhysic
 
         async Task InvokeIncomingTransportMessagesMutators(IIncomingPhysicalMessageContext context, Func<IIncomingPhysicalMessageContext, Task> next)
         {
-            var mutators = context.Builder.BuildAll<IMutateIncomingTransportMessages>();
+            var mutatorsRegisteredInDI = context.Builder.BuildAll<IMutateIncomingTransportMessages>();
```

Comment:
Since this is the only one of the 4 behaviors that has this local, can we instead just align this implementation to match the others, and have `context.Builder.BuildAll<IMutateIncomingTransportMessages>()` in the `foreach`?

Reply:
I'm fine with leaving this as-is on the hotfix, but let's go ahead and clean it up after the changes land in `develop`.